phpBB/includes/functions_upload.php

```diff
@@ -332,6 +332,26 @@ function move_file($destination, $overwrite = false, $skip_image_check = false,
 				@unlink($this->destination_file);
 			}
 
+			/**
+			* Event to modify avatars before writing to filesystem
+			*
+			* @event core.avatar_upload
+			* @var  string  filename     filename for upload
```

Comment:
Please use tabs instead of spaces to align text in doc blocks in accordance with our coding guidelines.